### PR TITLE
Generate changelog for 3.0.0-alpha3

### PR DESCRIPTION
**Pull request by kyukhin**
NO_DOC=no code changes
NO_TEST=no code changes
NO_CHANGELOG=no code changes

### CODE REVIEW

File: changelogs/3.0.0-alpha3.md [hunk 1]
```diff
@@ -0,0 +1,621 @@
...
+  `IPROTO_EXECUTE` or `IPROTO_PREPARE` requests. To revert to the old behavior
+  (no SQL access checks), use the `sql_priv` compat option (gh-8803).
+* The maximum length of `box.cfg{}` string parameters is now 512 instead of 256.
+* Added a new flag `tarantool.build.asan` that shows whether build
+  flag `ENABLE_ASAN` is set.
+* Added the `leader_name` field to `box.info.election` (gh-8931).
+
+#### Cfg
+
+* Implemented a way to set a table as box.cfg{} options value via
```
Comment (p7nov) [inline changelogs/3.0.0-alpha3.md:232]:
```suggestion
* Implemented a way to set a table as `box.cfg{}` options value via
```

File: changelogs/3.0.0-alpha3.md [hunk 2]
```diff
@@ -0,0 +1,621 @@
...
+  using the `\set continuation` command.
+
+### Debugger
+
+* Introduced readline support to tarantool debugger (gh-7738).
+
+### Space
+
+* Introduces space type: a new space definition field.
+* Introduces the fully temporary space type. It is the same as data-temporary
```
Comment (p7nov) [inline changelogs/3.0.0-alpha3.md:311]:
```suggestion
* Introduced space type: a new space definition field.
* Introduced the fully temporary space type. It is the same as data-temporary
```

File: changelogs/3.0.0-alpha3.md [hunk 3]
```diff
@@ -0,0 +1,621 @@
...
+* Fixed a bug because of which a dirty (not committed to WAL) DDL record could
+  be written to a snapshot and cause a recovery failure (gh-8530).
+* **[Breaking change]** The key `box.space._schema['cluster']` is renamed to
+  `'replicaset_uuid'`. That is not expected to be breaking because `_schema` is
+  an internal system space, but the key was visible in public and documented
+  (gh-5029).
+* **[Breaking change]** The table `box.info.cluster` is renamed to
+  `box.info.replicaset`. The behaviour can be reverted using the `compat` option
+  `box_info_cluster_meaning`
+  (https://tarantool.io/compat/box_info_cluster_meaning) (gh-5029).
```
Comment (p7nov) [inline changelogs/3.0.0-alpha3.md:370]:
```suggestion
  [box_info_cluster_meaning](https://tarantool.io/compat/box_info_cluster_meaning)
  (gh-5029).
```

**Review by p7nov — COMMENTED** [review]
Some fixes in suggestions

File: changelogs/3.0.0-alpha3.md [hunk 4]
```diff
@@ -0,0 +1,619 @@
...
+  a replica reconnect (gh-7991).
+* Fixed a possible crash on bootstrap with `box.cfg.bootstrap_strategy = 'auto'`
+  when some of the bootstrapping nodes were stopped (gh-8757).
+* Fixed a bug when `box.ctl.demote()` with `box.cfg{election_mode = 'off'}`
+  and an owned synchro queue could simply not do anything (gh-6860).
+
+### LuaJIT
+
+Backported patches from the vanilla LuaJIT trunk (gh-8069). The following issues
+were fixed as part of this activity:
```
[the remark this answers is not in the document]
Comment (kyukhin) [reply changelogs/3.0.0-alpha3.md]: Removed.

File: changelogs/3.0.0-alpha3.md [hunk 5]
```diff
@@ -0,0 +1,619 @@
...
+* Fixed canonicalization of +-0.0 keys for `IR_NEWREF`.
+* Fixed result truncation for `bit.rol` on x86_64 platforms.
+* Fixed saved bytecode encapsulated in ELF objects.
+* Fixed `lua_yield()` invocation inside C hooks.
+* Fixed memory chunk allocation beyond the memory limit.
+* Fixed TNEW load forwarding with instable types.
+* Fixed use-def analysis for `BC_VARG`, `BC_FUNCV`.
+Backported patches from the vanilla LuaJIT trunk (gh-8825). The following issues
+were fixed as part of this activity:
+
```
[the remark this answers is not in the document]
Comment (kyukhin) [reply changelogs/3.0.0-alpha3.md]: Removed.

File: changelogs/3.0.0-alpha3.md (hunk 2 above)
Comment (kyukhin) [reply changelogs/3.0.0-alpha3.md]: Fixed.

File: changelogs/3.0.0-alpha3.md (hunk 3 above)
Comment (kyukhin) [reply changelogs/3.0.0-alpha3.md]: Fixed.

**Review by p7nov — APPROVED** [review]

**Pull request merged**